### ISSUE TITLE
error loading package '': Extension file 'third_party/clang_toolchain/cc_configure_clang.bzl' has errors.

### ISSUE THREAD

**Issue by yangyuji12** [tensorflow/serving#851]
I am running a tensorflow serving container referring to [this](https://www.tensorflow.org/serving/docker) , all the previous steps are good, but in the last block I met some problems:  

> git clone --recurse-submodules https://github.com/tensorflow/serving
> cd serving/tensorflow
> ./configure
> cd ..
> bazel test tensorflow_serving/...  

I did not do the `./configure` line because that there is not a `serving/tensorflow` dir and I saw someone in another issue said there is no need to do that any more. So I directly does to the last line`bazel test tensorflow_serving/...`, but the error occurs:  

> root@15bb1c2766e3:/serving# bazel test tensorflow_serving/...
> ERROR: /root/.cache/bazel/_bazel_root/f8d1071c69ea316497c31e40fe01608c/external/org_tensorflow/third_party/clang_toolchain/cc_configure_clang.bzl:3:1: file '@bazel_tools//tools/cpp:cc_configure.bzl' does not contain symbol 'cc_autoconf_impl'.
> ERROR: error loading package '': Extension file 'third_party/clang_toolchain/cc_configure_clang.bzl' has errors.
> ERROR: error loading package '': Extension file 'third_party/clang_toolchain/cc_configure_clang.bzl' has errors.
> INFO: Elapsed time: 0.107s
> ERROR: Couldn't start the build. Unable to run tests. 

And in my container, the bazel version is 0.5.4:  

> root@15bb1c2766e3:/serving# bazel version
> Build label: 0.5.4
> Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
> Build time: Fri Aug 25 10:00:00 2017 (1503655200)
> Build timestamp: 1503655200
> Build timestamp as int: 1503655200  

Below is  my `bazel.rc`:

> root@15bb1c2766e3:/serving/tools# cat bazel.rc
> build:cuda --crosstool_top=@org_tensorflow//third_party/gpus/crosstool
> build:cuda --define=using_cuda=true --define=using_cuda_nvcc=true
> 
> build --force_python=py3
> build --python3_path=/usr/bin/python
> 
> build --action_env PYTHON_BIN_PATH="/usr/bin/python"
> 
> build --define PYTHON_BIN_PATH=/usr/bin/python
> 
> build --spawn_strategy=standalone --genrule_strategy=standalone
> test --spawn_strategy=standalone --genrule_strategy=standalone
> run --spawn_strategy=standalone --genrule_strategy=standalone
> 
> build --define=grpc_no_ares=true
> 
> \# TODO(b/69809703): Remove once no longer required for TensorFlow to build.
> build --copt=-DGEMMLOWP_ALLOW_SLOW_SCALAR_FALLBACK

I guess there may be something dismatch, but I can not find much information about the `third_party/clang_toolchain`.

**Comment by wangzhenhui1992** [comment]
I'm facing the same problem.

**Comment by guillaumegenthial** [comment]
possible merge (and fix) with #854.
TL;DR: change Dockerfile.devel BAZEL_VERSION to 0.12.0

**Comment by AswathKiruba** [comment]
We could also update bazel directly in the docker container by 
1.	Download bazel-<version>-installer-linux-x86_64.sh from https://github.com/bazelbuild/bazel/releases location
2.	chmod +x ./bazel-<version>-installer-linux-x86_64.sh
3.	./bazel-<version>-installer-linux-x86_64.sh

@RodMayen  Are using the tensorflow serving master (1.6) branch?

**Comment by RodMayen** [comment]
@AswathKiruba thanks a lot for your answer, actually I was wrong, the suggestion by @guillaumegenthial was the way to go for me! thanks a lot guys
